packages/core/src/shared/sam/sync.ts

```diff
@@ -686,6 +686,8 @@ class ProcessTerminal implements vscode.Pseudoterminal {
     public constructor(private readonly process: ChildProcess) {
         // Used in integration tests
         if (isAutomation()) {
+            // Disable because it is a test.
+            // eslint-disable-next-line aws-toolkits/no-console-log
             this.onDidWrite(text => console.log(text.trim()))
```

Comment:
This is fine for specific, intentionally cases. But it's good to have the rule generally enabled for tests.